src/diffusers/textual_inversion_utils.py

```diff
@@ -0,0 +1,101 @@
+import torch
+
+
+class TextualInversionMixin:
+    r"""
+    Mixin class for adding textual inversion tokens and embeddings to the tokenizer and text encoder with method:
+    - [`~TextualInversionMixin.load_textual_inversion_embeddings`]
+    - [`~TextualInversionMixin.add_textual_inversion_embedding`]
+
+    Class attributes:
+    - **textual_inversion_tokens** (`List[str]`): list of tokens added to the tokenizer's vocabulary and the text
+      encoder's embedding matrix
+    """
+    textual_inversion_tokens = []
```

Comment:
Do we need `textual_inversion_tokens`? If I'm reading correctly, it seems we just append to it. Class variables, especially on mixins, can get quite confusing as they're shared between all classes and instances which inherit from the mixin.

```py
class Mixin:
    var = []

class Base1(Mixin):
    ...

class Base2(Mixin):
    ...

instance1 = Base1()
instance2 = Base2()

print(Base1.var)
print(instance1.var)
print(instance2.var)

instance1.var.append(1)

print(Base1.var)
print(instance1.var)
print(instance2.var)

instance2.var.append(2)

print(Base1.var)
print(instance1.var)
print(instance2.var)
```

```
[]
[]
[]
[1]
[1]
[1]
[1, 2]
[1, 2]
[1, 2]
```

Reply:
No, we do not. Good catch. I removed this.